MyPerfectOnboardingApplication/Tests/MyPerfectOnboarding.App.Tests/Services/ListCacheTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Management.Instrumentation;
+using System.Threading.Tasks;
+using MyPerfectOnboarding.Contracts.Database;
+using MyPerfectOnboarding.Contracts.Models;
+using MyPerfectOnboarding.Contracts.Services.Generators;
+using MyPerfectOnboarding.Contracts.Services.ListItems;
+using MyPerfectOnboarding.Contracts.Services.Location;
+using MyPerfectOnboarding.Dependency;
+using MyPerfectOnboarding.Dependency.Containers;
+using MyPerfectOnboarding.Services.Services;
+using MyPerfectOnboarding.Tests.Utils.Builders;
+using NSubstitute;
+using NUnit.Framework;
+using Unity;
+
+namespace MyPerfectOnboarding.App.Tests.Services
```

Comment:
```suggestion
namespace MyPerfectOnboarding.Application.Tests.Services
```
https://github.com/HanaNavratilova/kentico-onboarding-cs/pull/4#discussion_r261535994 💩

Reply:
You might actually want an extra folder here (`.Services.Services`), however, I'm more than willing to overlook this one 👀